src/test/java/com/microsoft/graph/BaseClient.java

```diff
@@ -10,10 +13,10 @@
 /**
  * Default client implementation.
  */
-class BaseClient implements IBaseClient{
+public class BaseClient implements IBaseClient {
```

Comment:
BaseClient should ideally be non-public. Then the service libs will simply implement `IBaseClient`

Reply:
Moved this to the testing module.